dsl/dsl-defaults.js

```diff
@@ -1,56 +1,58 @@
-/* jshint node: true */
+/* eslint-env node */
 'use strict';
 
 const LAYOUT_PROPERTIES = [
-  "fill",
-  "fit",
-  "horizontal",
-  "nowrap",
-  "vertical",
-  "wrap",
-  { name: 'align', values: ["start", "end", "stretch", "center", "baseline"] },
-  { name: 'justify', values: ["start", "end", "center", "between", "around"] },
+  'fill',
+  'fit',
+  'horizontal',
+  'nowrap',
+  'vertical',
+  'wrap',
+  { name: 'align', values: ['start', 'end', 'stretch', 'center', 'baseline'] },
+  { name: 'align-content', values: ['start', 'end', 'stretch', 'center', 'around', 'between'] },
+  { name: 'align-self', values: ['start', 'end', 'stretch', 'center', 'baseline'] },
```

Comment:
These two lines are the only additions of consequence. The rest are making eslint happy